public/app/plugins/datasource/grafana-azure-monitor-datasource/components/ResourcePicker/utils.ts

```diff
@@ -0,0 +1,28 @@
+import { Row, RowGroup } from './types';
+
+const RESOURCE_URI_REGEX = /\/subscriptions\/(?<subscriptionID>.+)\/resourceGroups\/(?<resourceGroup>.+)\/providers\/(?<cloud>.+)/;
```

Comment:
oh hey here it is... again, if these are in fact all the same lets use some kind of util somewhere